### ISSUE TITLE
cordova.plugins is undefined

### ISSUE THREAD

**Issue by DocGreenRob** [Telerik-Verified-Plugins/LocalNotification#17]
I just added this plugin to my appbuilder web ide project and tried to simply use `//cordova.plugins.notification.local.schedule({
            //    id: 1,
            //    title: 'I will bother you every minute',
            //    text: '.. until you cancel all notifications',
            //    sound: null,
            //    every: 'minute',
            //    autoClear: false,
            //    at: new Date(new Date().getTime() + 10 * 1000)
            //});`

and I get the error above.

**Comment by EddyVerbruggen** [comment]
Did you wait for `deviceready` to fire?

**Comment by DocGreenRob** [comment]
yes. I added the plugin, then added this code on a button click event, so well after the deviceready fired...

**Comment by EddyVerbruggen** [comment]
And you're running it in a webbrowser?

**Comment by DocGreenRob** [comment]
Yes

**Comment by EddyVerbruggen** [comment]
Browsers don't implement the Local Notification API so you really need to use a device / simulator.

**Comment by DocGreenRob** [comment]
Incorrect. I am using the WebBrowser IDE to develop the app but testing Local Notification API on my device as I am well aware of this limitation.  Nevertheless, during debugging I get cordova.plugins is undefined... Please do not close this ticket as I need HELP!

**Comment by EddyVerbruggen** [comment]
I asked "And you're running it in a webbrowser?" and you said "Yes" so I think my respons was not incorrect. It's just that you thought I meant "IDE" whereas I meant "app".

Share your app with me and I may take a look.

**Comment by DocGreenRob** [comment]
Ok.
[Notepad.zip](https://github.com/Telerik-Verified-Plugins/LocalNotification/files/433148/Notepad.zip)
Here you go...

**Comment by EddyVerbruggen** [comment]
This code has no method tied to a button and no active `cordova.plugins.noti..` code (commented) so it's hard for me to tell what exactly is/was live while you observe the error.

That being said the code looks reasonable to me. Do other plugins (like Toast which you've included as well) work?

**Comment by DocGreenRob** [comment]
Thanks for that second question as Toast gives same error, plugin undefined.  Notice that the cordova.plugins.notification.local.schedule is called in the app.js line 48. Check it out and you will see it is after deviceready... let me know your findings...

**Comment by DocGreenRob** [comment]
@EddyVerbruggen any update on the file I shared with you? (Did you discover anything helpful?)
